lib/moduleConfig.js

```diff
@@ -5,8 +5,13 @@ var cache = require("./cache"); //use requires caching to have a singleton
 var getConfigPath = require("./getConfigPath"); //use requires caching to have a singleton
 var path = require("path");
 
-module.exports = function moduleConfig(paths, loadPathFunction) {
-	var startDir = path.dirname(module.parent.parent.filename);
+module.exports = function moduleConfig(startDir, paths, loadPathFunction) {
```

Comment:
make `startDir` optional and pass it as the last parameter - then it does not break existing usages
```suggestion
module.exports = function moduleConfig(paths, loadPathFunction, startDir=null) {
```

Reply:
I agree this is a mess and I see that loadPathFunction is optional. Why not write your own module for moduleconfig?

Reply:
I think we just wanted to avoid releasing this package under our scope. 
@hilleer Do you have any thoughts on this?

Reply:
Not really. whatever you find better/more suitable. Optimally I would simply pass an object to the function instead of optional arguments where the object has the optional options 🙂